docs/source/examples.rst

```diff
@@ -1,10 +1,295 @@
+with the use of ns3-ai. You need to transfer both of these directories in the appropriate locations by running the
+following commands:
+
+.. code-block:: bash
+
+    cp -r $REINFORCED_LIB/examples/ns-3/rlib-sim $YOUR_NS3_PATH/scratch/
+    cp -r $REINFORCED_LIB/examples/ns-3/rlib-wifi-manager $YOUR_NS3_PATH/contrib/
+
+.. note::
+    To learn more about adding contrib modules to ns-3, visit
+    the `ns-3 manual <https://www.nsnam.org/docs/manual/html/new-modules.html>`_.
+
+
+.. _rlib-sim:
+
+Simulation scenario
+===================
+
+
+ns-3 (C++) end
+--------------
+
+We supply an example scenario ``rlib-sim\sim.cc`` to test the rate adaptation manager. The scenario is highly customizable but the key points
+are that there is one access point AP and a variable number (``--nWifi``) of stations STA; there is an uplink, saturated
+comunication (from AP to STAs) and the AP is in clear line of sight from all the STAs; All the STAs are in the point of 0m
+and the AP can be either in 0m as well or in some distance (``--initialPosition``) from the STAs. The AP can also be moving
+with a constant velocity (``--velocity``) to simulate dynamic scenarios. Other assumptions from the simulation are the
+log-distance propagation `loss model <https://www.nsnam.org/docs/models/html/propagation.html>`_ and AMPDU frames aggregation.
+  
+  Changable simulation parameters:
+  
+  * Duration of the simulation; excluding warmup stage (s) ``--simulationTime``, default to 20 s
+  * Duration of the warmup stage (s) - a time for the simulator to enable all the mechanisms before the traffic begins ``--warmupTime``, default to 2 s
+  * Time interval between successive measurements (s) ``--logEvery``, default to 1 s
+  * Simulation Seed ``--RngRun``
+  
+---------------
+
+  * Aggregated traffic generators data rate (Mb/s) ``--dataRate``, default to 125 Mb/s
+  * Channel width (MHz) ``--channelWidth``, default to 20 MHz
+  * Shortest guard interval (ns) ``--minGI``, default to 3200 ns
+  * Rate adaptation manager ``--wifiManager``, default to ``"ns3::RLibWifiManager"``, meaning that the manager is on the Reinforced-lib side
+  
+---------------
+
+  * Relative path where the simulation output file will be saved in the CSV format ``--csvPath``, default to ``""``, meaning no save at all
+  * Name of the Wi-Fi manager in CSV ``wifiManagerName``, default to ``"RLib"``
+  * Relative path where the PCAP file from the AP will be saved ``--pcapPath``, default to ``""``, meaning no pcap at all
+
+
+Reinforced-lib (python) end
 ---------------------------
+
+The provided Rate Adaptation manager is implemented in the file ``$REINFORCED_LIB/examples/ns-3/main.py``. Here we specify the
+comunication with the ns-3 simulator by defining the environment observation space and the action space, we create the ``RLib``
+agent, we provide the listening loop which reacts to the incomming (aggregated) frames by responding with the appropriate MCS
```

Comment:
Zastanawiam się nad tym określeniem "listening loop" czy to dobrze brzmi. W tematyce RL ja się spotkałem z określeniem "agent-environment interaction loop", więc w razie czego można na to zamienić.

Reply:
Okey, ja się bardziej skoncentrowałej jako event-loop i reakcjana zdarzenia, ale masz racje że w kontekście RL'a "agent-environment interaction loop" brzmi lepiej